src/main/java/com/bean00/OverriddenMessageController.java

```diff
@@ -0,0 +1,17 @@
+package com.bean00;
+
+import java.io.BufferedReader;
+import java.io.PrintWriter;
+
+public class OverriddenMessageController extends MessageController {
```

Comment:
This exists only as a stub for testing. It should be in `test/main/java` not `src/main/java`.

Reply:
This needs a better name. What are you overriding and why? Use your answer to that question to choose a better name.

Reply:
Class moved and renamed to __